shared/actions/platform-specific/index.native.js

```diff
@@ -364,6 +364,13 @@ const editAvatar = () =>
     })
   })
 
+const openAppStore = () =>
+  Linking.openURL(
+    isAndroid
+      ? 'http://play.google.com/store/apps/details?id=io.keybase.ossifrage'
```

Comment:
https?

Reply:
They say http. https://developer.android.com/distribute/marketing-tools/linking-to-google-play

Reply:
I wonder if that's intentional It gets redirected to https in browser. I'm not sure how it works on Android; I guess if its skips browser and goes to the app store directly it's fine.